app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Well done adding the associations however according to the ERB Diagram it should be:
`belongs_to :author, class_name: 'User'`

Reply:
Solved :+1: